ROMFS/px4fmu_common/init.d/rc.interface

```diff
@@ -119,10 +120,43 @@ then
 	then
 		if fmu mode_pwm
 		then
-			mixer load /dev/pwm_output1 $MIXER_AUX_FILE
+			mixer load $OUTPUT_AUX_DEV $MIXER_AUX_FILE
 		else
 			tone_alarm $TUNE_ERR
 		fi
+
+		if [ $PWM_AUX_OUT != none ]
```

Comment:
Is the NuttX default of an unset variable none? Else please add ```set PWM_AUX_OUT none``` in the top level rcS file where PWM_OUT is set.